server/graphql/v2/object/Tier.js

```diff
@@ -62,6 +64,36 @@ export const Tier = new GraphQLObjectType({
           return { nodes: result.rows, totalCount: result.count, limit: args.limit, offset: args.offset };
         },
       },
+      amount: {
+        type: Amount,
+        resolve(tier) {
+          return { value: tier.amount, currency: tier.currency };
+        },
+      },
+      interval: {
+        type: GraphQLString,
+        resolve(tier) {
+          return tier.interval;
+        },
+      },
+      presets: {
+        type: new GraphQLList(GraphQLInt),
+        resolve(tier) {
+          return tier.presets;
+        },
+      },
+      amountType: {
+        type: GraphQLString,
```

Comment:
- Should be not nullable
- Would be nice to create an enum for this one

```suggestion
        type: new GraphQLNonNull(GraphQLString),
```